staging/src/k8s.io/legacy-cloud-providers/gce/gce_disks.go

```diff
@@ -990,3 +1038,22 @@ func isGCEError(err error, reason string) bool {
 	}
 	return false
 }
+
+// verifyDisksAttachedToNode takes in an array to disks that should be attached to an instance, and the
+// array of disks actually attached to it. It returns a verifying if the disks are actually attached.
+func verifyDisksAttachedToNode(disksToVerify []string, disksActuallyAttached []*compute.AttachedDisk) map[string]bool {
+	verifiedDisks := make(map[string]bool)
+	disksActuallyAttachedSet := make(map[string]struct{})
```

Comment:
use `sets.String`. you can directly change the list into a Set and use set `contains` to check for existence

Reply:
I don't think sets.String can easily be used, because the array passed in is not an array of strings but an array of AttachedDisks item. I would have to apply a map that gets the name of each AttachedDisk but the cost of that is another loop which will only make the code more confusing.

Reply:
use the same loop as you have before (+1 for using sets.String)